.github/workflows/azure-sdk-tools.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Azure SDK Tools
+
+on:
+  workflow_dispatch:
+  push:
+    branches: []
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-22.04
```

Comment:
Up to you, but I've been using `ubuntu-latest`.  I believe the majority of users (in general) use the `-latest` images, so I'm willing to take the risk of breaking, over the cost of updating all our workflows every major Ubuntu version.

Reply:
I'm also ok with trying to use latest and only pinning if any when necessary to workaround an specific image issue.